client/signup/steps/domains/style.scss

```diff
@@ -3,12 +3,27 @@
 
 .domains__step-section-wrapper {
 	margin: 0 auto;
-	max-width: 720px;
+	width: 100%;
 }
 
 .is-section-signup .domains__step-content {
 	margin-bottom: 50px;
 
+	& .use-my-domain__page-heading {
+		padding-left: 24px;
+		padding-right: 24px;
+	}
+
+	& .use-my-domain,
+	& .domain-transfer-or-connect__content {
+		box-shadow: none;
+
```

Comment:
```suggestion
	& .use-my-domain,
	& .connect-domain-step,
	& .domain-transfer-or-connect__content {
		background: transparent;
		box-shadow: none;
```

Reply:
That will fix both the background color and the `box-shadow` prop on the transfer option which I referred to in the review.